test_scripts/API/Restructuring_OnResetTimeout/002_BC_OnResetTimeout_Success_flow_without_resetPeriod.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL
```

Comment:
please mention what is the value of defaultTimeout specified in smartDeviceLink.ini. if defaultTimeout =5 seconds, it may timeout before 11 seconds, thus the response from HMI at 11s will be ingored